ert/data/record/_transformation.py

```diff
@@ -228,64 +316,82 @@ async def _save_record_to_file(record: Record, location: Path, mime: str) -> Non
             await ft.write(get_serializer(mime).encode(record.data))
     else:
         async with aiofiles.open(str(location), mode="wb") as fb:
-            await fb.write(record.data)  # type: ignore
+            assert isinstance(record.data, bytes)  # mypy
+            await fb.write(record.data)
 
 
-class RecordTreeTransformation(RecordTransformation):
+class RecordTreeTransformation(SerializationTransformation):
```

Comment:
double check the name, whether or not it make sense

Reply:
@sondreso I don't have a good suggestion for name here… will you help me out?

Reply:
Yhea, thats not an easy one.. TreeSerializationTransformation maybe? :shrug: